intl_translation_arb/test/intl_translation_arb_test.dart

```diff
@@ -1,12 +1,40 @@
 import 'package:intl_translation_arb/intl_translation_arb.dart';
+import 'package:intl_translation_format/src/translation_format.dart';
 import 'package:intl_translation_format/test_utils.dart';
 import 'package:test/test.dart';
 
 void main() {
   group('Arb Format -', () {
-    testFormatParserWithDefaultMessages(
-      ArbFormat(),
-      simpleMessage: '''
+    testFormat(ArbDefaultFormatTester());
+
+    test('File name', () {
+      final template = MockTemplate('intl', {});
+      template.lastModified = null;
+      final result = ArbFormat().generateTemplateFiles(template);
+
+      expect(result.first.name, 'intl_en.arb');
+    });
+
+    test('Locale fr', () {
+      final template = MockTemplate('intl', {}, locale: 'fr');
+      template.lastModified = null;
+      final result = ArbFormat().generateTemplateFiles(template);
+
+      expect(result.first.name, 'intl_fr.arb');
+      expect(result.first.contents, '''
+{
+  "@@locale": "fr"
+}''');
+    });
+  });
+}
+
+class ArbDefaultFormatTester extends MonolingualFormatTester {
```

Comment:
This is nice, but should definitely have comments. I'd put some here, because I think people will look at this as an example to copy/understand more than they will look at the superclass. I mean, it should have comments too, but you want some here.